src/components/atoms/ArticleCard/index.js

```diff
@@ -24,12 +24,16 @@ const ArticleCard = ({
 
 ArticleCard.propTypes = {
   title: PropTypes.string.isRequired,
-  description: PropTypes.string.isRequired,
+  description: PropTypes.string,
   name: PropTypes.string.isRequired,
   time: PropTypes.string.isRequired,
   img: PropTypes.string.isRequired,
-  id: PropTypes.number.isRequired,
+  id: PropTypes.number,
   history: PropTypes.shape({ push: PropTypes.func.isRequired }).isRequired
 };
+ArticleCard.defaultProps = {
+  description: '',
+  id: NaN
```

Comment:
Good Job 👍
However, since the *id* prop expects a number, I think it will be better to initialize it with a *0* rather than *NaN*. This was unforeseen bugs are prevented.

Reply:
Alright, thanks